internal/util/projutil/exec.go

```diff
@@ -22,11 +22,13 @@ import (
 	"strings"
 
 	homedir "github.com/mitchellh/go-homedir"
+	log "github.com/sirupsen/logrus"
 )
 
 func ExecCmd(cmd *exec.Cmd) error {
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
+	log.Debugf("Running %#v", cmd.Args)
```

Comment:
I think we need to remove this.

Reply:
That's actually the code that prints out the arguments to a command line, but it only produces output if the `logrus` level is set to debug at launch.

Reply:
@shawn-hurley does that make sense?